bench/wtperf/wtperf.c

```diff
@@ -456,6 +486,79 @@ worker(void *arg)
 		goto err;
 	}
 
+	/* Setup for truncate */
+	if (thread->workload->truncate != 0) {
+
+		STAILQ_INIT(&truncate_stone_head);
+
+		cursor = cursors[cfg->icount % cfg->table_count];
```

Comment:
AFAI can see this is the only place you reference the multi-table configuration supported by ```wtperf```. I suspect this won't work properly all the time.

I'd recommend excluding this truncate functionality from the multi-table configuration for now during configuration parsing phase.